pyanaconda/modules/boss/module_manager/start_modules.py

```diff
@@ -40,17 +38,22 @@ class StartModulesTask(Task):
     method StartServiceByName is called.
     """
 
-    def __init__(self, message_bus, module_names, addons_enabled):
+    def __init__(self, message_bus, activitable, forbidden, optional):
```

Comment:
Should the var be named `activable` instead?

Reply:
It should be `activatable`. Good catch!

Reply:
Fixed.